src/screens/VisualTests/BuildEyebrow.stories.ts

```diff
@@ -80,26 +93,12 @@ export const Snapshot: Story = {
 export const Complete: Story = {
   args: {
     runningBuild: {
-      step: "complete",
+      ...INITIAL_BUILD_PAYLOAD,
+      currentStep: "complete",
       buildProgressPercentage: 100,
     },
   },
   parameters: withFigmaDesign(
     "https://www.figma.com/file/GFEbCgCVDtbZhngULbw2gP/Visual-testing-in-Storybook?type=design&node-id=2303-375342&mode=design&t=vlcsXN2x67tQaQdy-0"
   ),
 };
-
-export const Latest: Story = {
-  // Could not find this state in the figma file, but it is in this video: https://chromaticqa.slack.com/archives/C051TQR6QLC/p1692372058786929?thread_ts=1692354384.352659&cid=C051TQR6QLC
-  // parameters: withFigmaDesign(
-  // ),
-};
-
-export const LatestNeedToUpdate: Story = {
-  args: {
-    switchToNextBuild: undefined,
-  },
-  parameters: withFigmaDesign(
-    "https://www.figma.com/file/GFEbCgCVDtbZhngULbw2gP/Visual-testing-in-Storybook?type=design&node-id=2127-448761&mode=design&t=70EtYCn1H7hB8PAk-0"
-  ),
-};
```

Comment:
We still need these two stories I think -- you updated the code for them after all:

<img width="773" alt="image" src="https://github.com/chromaui/addon-visual-tests/assets/132554/d7667357-086d-47a7-8ce6-ed9de77ee788">